Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
+The choice of calendar defines the set of dates (year-month-day combinations) which are permitted, and therefore it specifies the number of days between the times of 0:0:0 (midnight) on any two dates. Date/times which are not permitted in a given calendar are prohibited in both the encoded time coordinate values, and in the reference date/time string.<br>
+
+**Requirements**
 
-The CF Conventions provides a complete [list of variable attributes](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#attribute-appendix) it covers.
-Here we are covering only the ones we believe are more critical. It is also worth to point out that lots of these attributes are not required but highly recommended for backwards compatibility with COARDS. However, they are usually expected for the files to be considered CF compliants, whenever they are applicable.
+* The attributes calendar, month_lengths, leap_year, and leap_month may only be attached to time coordinate variables.
+* The standardized values (case insensitive) of the calendar attribute are standard, gregorian (deprecated), proleptic_gregorian, noleap, 365_day, all_leap, 366_day, 360_day, julian, and none. If the calendar attribute is given a non-standard value, then the attribute month_lengths is required, along with leap_year and leap_month as appropriate.
+* The type of the month_lengths attribute must be an integer array of size 12.
+* The values of the leap_month attribute must be in the range 1-12.
+* The values of the leap_year and leap_month attributes are integer scalars.
 
-```{dropdown} units 
-CF - required
-Latitude, longitude, and time coordinates are identified solely by the value of their units attribute. Vertical coordinates with units of pressure may also be identified by the units attribute. Other vertical coordinates must use the attribute positive which determines whether the direction of increasing coordinate value is up or down. Because identification of a coordinate type by its units involves the use of an external software package [UDUNITS](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#UDUNITS), we provide the optional attribute axis for a direct identification of coordinates that correspond to latitude, longitude, vertical, or time axes.
+**Recommendations**
+
+* The value standard should be used instead of gregorian in the calendar attribute.
+* The attribute leap_month should not appear unless the attribute leap_year is present.
+* The time coordinate should not cross the date 1582-10-15 when the default mixed Gregorian/Julian calendar is in use.
```

Comment:
Should we put a callout here for how to handle long-run mfdatasets? That's more of a Big Data concern, just wondering.

Reply:
If I'm interpreting this correctly, you mean some advice on how to have a consistent time axis for data, like a model simulation, that spans across different files.
If that's the case then I think that this should be definitely covered in the create dataset section of this book.